jenkins/flow/pr/edx-platform-python-unittests-pr.groovy

```diff
@@ -92,7 +93,9 @@ guard{
       lms_unit_3.result.toString() == 'SUCCESS' &&
       lms_unit_4.result.toString() == 'SUCCESS' &&
       cms_unit.result.toString() == 'SUCCESS' &&
-      commonlib_unit.result.toString() == 'SUCCESS')
+      commonlib_unit.result.toString() == 'SUCCESS' &&
+      runCoverage
```

Comment:
Just curious since I don't know Groovy: Does `runCoverage` need to be compared to "true" or "false"? Or does Groovy know to convert the string into a boolean?

Reply:
Good catch. The value is a boolean in the DSL, but we lose that by passing it as an environment varaible. Casting it as a boolean now.